.clomonitor.yml

```diff
@@ -0,0 +1,9 @@
+# CLOMonitor metadata file
+# This file must be located at the root of the repository
+
+# Checks exemptions
+exemptions:
+  - check: license_scanning # Check identifier (see https://github.com/cncf/clomonitor/blob/main/docs/checks.md#exemptions)
+    reason: "There are currently no plans moving forward to implement FOSSA or Snyk for scanning purposes." # Justification of this exemption (mandatory, it will be displayed on the UI)
+  - check: artifacthub_badge
+    reason: "This repository has no items that should be added to Artifact Hub."
```

Comment:
The registry operator could be added to artifact hub as an OLM operator but there are no plans to add it to artifact hub at this time. devfile/api#1009 is the plans to add the helm chart located under [devfile/registry-support](https://github.com/devfile/registry-support/tree/35722ec1cc6e5599afe88983d7ead4145d87c8c5/deploy/chart/devfile-registry) iirc.

Reply:
Going to remove the artifact hub check as Michael pointed out the OLM operator could be added but we don't plan to.